runtime/commonMain/src/kotlinx/serialization/json/internal/JsonReader.kt

```diff
@@ -9,6 +9,8 @@ import kotlinx.serialization.json.internal.EscapeCharMappings.ESCAPE_2_CHAR
 import kotlin.jvm.*
 import kotlin.native.concurrent.*
 
+internal const val lenientHint = "Use 'JsonConfiguration.isLenient = true' to accept non-compliant JSON"
```

Comment:
Not sure how it end up here, but the hint is incorrect now: it mentions deprecated JsonConfiguration.

Also make sure that other hints (e.g. UnknownKeyException with 'You can enable 'JsonConfiguration.ignoreUnknownKeys') are also up-to-date